tests/transformer/test_try.py

```diff
@@ -47,6 +48,31 @@ def test_RestrictingNodeTransformer__visit_Try__2(
     ])
 
 
+TRY_EXCEPT_STAR = """
+def try_except_star(m):
+    try:
+        m('try')
+        raise ExceptionGroup("group", [IndentationError('f1'), ValueError(65)])
+    except* IndentationError:
+        m('IndentetionError')
+    except* ValueError:
+        m('ValueError')
+"""
+
+
+def test_RestrictingNodeTransformer__visit_Try__3(mocker):
```

Comment:
There should be a pytest marker for Python 3.11 and up

Reply:
Fixed